iml-wire-types/src/lib.rs

```diff
@@ -1895,12 +1895,50 @@ pub struct LustreClient {
     pub mountpoints: Vec<String>,
 }
 
+#[cfg_attr(feature = "graphql", derive(juniper::GraphQLEnum))]
+#[cfg_attr(feature = "postgres-interop", derive(sqlx::Type))]
+#[cfg_attr(feature = "postgres-interop", sqlx(rename = "fs_type"))]
+#[cfg_attr(feature = "postgres-interop", sqlx(rename_all = "lowercase"))]
+#[derive(PartialEq, Eq, Clone, Debug, serde::Serialize, serde::Deserialize, Ord, PartialOrd)]
+pub enum FsType {
+    Zfs,
+    Ldiskfs,
+}
+
+impl From<&str> for FsType {
+    fn from(x: &str) -> Self {
+        match x {
+            "ldiskfs" => Self::Ldiskfs,
+            "zfs" => Self::Zfs,
+            _ => Self::Ldiskfs,
```

Comment:
I don't think we should assume that any string == ldiskfs. Where is this being used?

Reply:
This is now matching on both zfs and ldiskfs and if neither are a match then it won't be used.